From e701fa4adf72460f1300cbe4b968e30ff05490ff Mon Sep 17 00:00:00 2001
From: Ashish Kalra <akalra@marvell.com>
Date: Tue, 21 May 2019 21:16:36 +0530
Subject: [PATCH 248/386] octeontx2-gti: Fix modprobe warnings with
 gti_watchdog driver

Change-Id: I0ebbc51174bec60cf8a56326afdf4754a68897ef
Reviewed-on: https://sj1git1.cavium.com/9364
Tested-by: sa_ip-sw-jenkins <sa_ip-sw-jenkins@marvell.com>
Reviewed-by: Stefan Chulski <Stefan.Chulski@cavium.com>
Reviewed-by: Sunil Kovvuri Goutham <Sunil.Goutham@cavium.com>
[RH: Original patch taken from marvell 88F3720 board support SDK 10.0-PR2003]
Signed-off-by: Ruiqiang Hao <Ruiqiang.Hao@windriver.com>
---
 drivers/soc/marvell/gti/gti_watchdog.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/soc/marvell/gti/gti_watchdog.c b/drivers/soc/marvell/gti/gti_watchdog.c
index eadee0a6949b..6300e019f440 100644
--- a/drivers/soc/marvell/gti/gti_watchdog.c
+++ b/drivers/soc/marvell/gti/gti_watchdog.c
@@ -177,7 +177,7 @@ static struct miscdevice gti_wdog_miscdevice = {
 	.fops = &gti_wdog_fops,
 };
 
-static int __init gti_wdog_probe(struct platform_device *pdev)
+static int gti_wdog_probe(struct platform_device *pdev)
 {
 	struct resource *r;
 	int ret_val;
@@ -208,7 +208,7 @@ static int __init gti_wdog_probe(struct platform_device *pdev)
 	return 0;
 }
 
-static int __exit gti_wdog_remove(struct platform_device *pdev)
+static int gti_wdog_remove(struct platform_device *pdev)
 {
 	pr_info("gti wdog platform driver exit\n");
 	if (g_gti_devmem)
-- 
2.17.1

